contracts/libs/StringStorage.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+library StringStorage {
+    struct Wrapper {
+        string value;
+    }
+
+    function store(string memory str, uint256 slot) internal {
```

Comment:
`store(uint256 slot, string memory str)` for consistency?

Reply:
Also `store` name is a bit too generic and might be conflicting with the app functions?

Reply:
Let's use `bytes32` instead of `uint256` for slots as we do in other places.

Reply:
`_setString` instead of `store` for consistency and since it's an internal method?